source/recipes/faq.md

```diff
@@ -2,6 +2,33 @@
 
 ## Nix
 
+### Should I enable flakes?
+
+You have to judge for yourself based on your needs.
+
+Flakes and the `nix` command suite bring multiple improvements that are relevant for both software users and package authors:
+- The new command line interface, together with flakes, makes dealing with existing packages significantly more convenient.
+- The constraints imposed on flakes strengthen reproducibility by default, and enable various performance improvements when interacting with a large Nix package repository like {term}`Nixpkgs`.
+- Flake references allow for easier handling of version upgrades for existing packages or project dependencies.
+- The flake schema helps with composing Nix projects from multiple sources in an ordered fashion.
+
+Other than that, and below the surface of the flake schema, Nix and the Nix language work the exactly same in both cases.
+The same level of reproducibility can be achieved  with or without flakes.
```

Comment:
```suggestion
In principle, same level of reproducibility can be achieved with or without flakes.
```

@zmitchell given the significant increase in convenience is stated at the beginning, this should be acceptable. What do you think?

Reply:
```suggestion
In principle, the same level of reproducibility can be achieved  with or without flakes.
```

That sounds fine, you're just missing a word here.